lib/ybe.gi

```diff
@@ -774,7 +774,7 @@ InstallMethod(IsIndecomposable,
   [ IsYB ],
   function(obj)
     if not IsInvolutive(obj) then
-      return fail;
+      Error("the solutions of the YBE is not involutive");
```

Comment:
Thank you @ilafly! Although, this text mixes plural and singular: "solutions ... is"

Reply:
It might be also nice to construct a test for this example - the CI complains that this is not covered by tests.